common/flavorization/wc_title_holders.txt

```diff
@@ -86,128 +134,1876 @@ warlord_female = {
+}
+empire_feudal_highelf = {
+	type = title
+	tier = empire
+	priority = 400
+    governments = { feudal_government }
+	cultures = { high_elf blood_elf }
+}
+
+### Highborne / Nightbornes ###
+# Nighborne specific
+count_feudal_male_nightborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne }
+}
+count_feudal_female_nightborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne }
+}
+county_feudal_nightborne = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	cultures = { nightborne }
+    governments = { feudal_government theocracy_government }
+}
+
+# Feudal
+duke_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 400
+	top_liege = no
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+duke_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 400
+	top_liege = no
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+duchy_feudal_highborne = {
+	type = title
+	tier = duchy
+	priority = 400
+	cultures = { nightborne highborne night_elf }
+    governments = { feudal_government theocracy_government }
+}
+king_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 400
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+king_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 400
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+kingdom_feudal_highborne = {
+	type = title
+	tier = kingdom
+	priority = 400
+	cultures = { nightborne highborne night_elf }
+    governments = { feudal_government theocracy_government }
+}
+emperor_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
+}
+emperor_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	governments = { feudal_government theocracy_government }
+	cultures = { nightborne highborne night_elf }
 }
 
-# King
-grand_prince_nightborne_male = {
+### Nagas ###
+count_feudal_male_naga = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 400
+	governments = { feudal_government tribal_government }
+	cultures = { naga }}
```

Comment:
And revert https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/71fb75d0d79596255f9420c318bfccaefbc7db5a, that's not fix.
```suggestion
	cultures = { naga }
```

Reply:
Also, fix another bracket errors, that's not the only one.

Reply:
Also, fix errors from `Documents\Paradox Interactive\Crusader Kings III\logs\error.log`

Reply:
Done ? Maybe.